### PR TITLE
[MBL-17262][Student] To Do details screen

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan:  
- same functionality as in the Flutter version 
- slightly different design (check Figma)
- Edit and delete will be implemented in another ticket

refs: MBL-17262
affects: Student
release note: none

## Checklist

- [ ] Follow-up e2e test ticket created or not needed
- [ ] Run E2E test suite
- [ ] Tested in dark mode
- [ ] Tested in light mode
- [ ] A11y checked

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: libs/pandautils/src/main/java/com/instructure/pandautils/features/todo/details/composables/OverflowMenu.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import com.instructure.pandautils.R
+
+@Composable
+fun OverflowMenu(content: @Composable () -> Unit) {
```
Comment (tamaskozmer) [inline libs/pandautils/src/main/java/com/instructure/pandautils/features/todo/details/composables/OverflowMenu.kt:36]: I would move this out to a common package where we will store our common Composables that can be reused by multiple screens. Also shouldn't this have a modifier parameter that is passed to the `DropdownMenu`?

File: libs/pandautils/src/main/java/com/instructure/pandautils/features/todo/details/composables/ToDoScreen.kt [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                }
+            }
+        )
+    }
+}
+
+@ExperimentalFoundationApi
+@Preview(showBackground = true)
+@Composable
+fun CalendarPreview() {
```
Comment (tamaskozmer) [inline libs/pandautils/src/main/java/com/instructure/pandautils/features/todo/details/composables/ToDoScreen.kt:172]: I would rename this to ToDoPreview.

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**